systemtest/src/test/java/io/strimzi/systemtest/TopicST.java

```diff
@@ -164,6 +164,36 @@ void testBigAmountOfTopicsCreatingViaKafka() {
         }
     }
 
+    @Test
+    void testTopicModificationOfReplicationFactor() throws InterruptedException {
+        String topicName = "topic-with-changed-replication";
+
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 2, 1).done();
+
+        testMethodResources().topic(CLUSTER_NAME, topicName)
+                .editSpec()
+                    .withReplicas(2)
+                .endSpec()
+                .done();
+
+        Thread.sleep(1000);
```

Comment:
I think you should wait here for the topic to be created by the TO first instead of relying on a timeout.  Either from Kafka or for example from `KAfkaTopic` status again.

Reply:
Changed.